spec/services/notification_service/course_published_spec.rb

```diff
@@ -47,12 +49,33 @@ module NotificationService
         allow(course).to receive(:findable?).and_return(findable)
       end
 
+      describe "when course is in the current cycle" do
```

Comment:
```suggestion
      context "with a course that is in the current cycle" do
```

Reply:
I've change to `context` and updated the descriptions